python/py-zstd/Portfile

```diff
@@ -29,9 +30,11 @@ if {${name} ne ${subport}} {
 
     depends_lib-append \
                     port:zstd
+    patchfiles-append \
+                   patch-disablecffi.diff
```

Comment:
the name of this patchfile does not correspond to what it's doing and thus causes confusion - please rename